matter/wifi/rs911x/rsi_if.c

```diff
@@ -398,7 +419,7 @@ void wfx_rsi_task(void *arg) {
             | 0,
         pdTRUE,  /* Clear the bits */
         pdFALSE, /* Wait for any bit */
-        pdMS_TO_TICKS(DELAY_250MS));
+        pdMS_TO_TICKS(250)); /* 250 mSec */
```

Comment:
A define should be used here.

Reply:
for pdMS_TO_TICKS i believe it is acceptable to use 250 directly.

Reply:
Kept 250 only for delay.